classes/01class/exercises/c01-git03/mendoncaj/README.md

```diff
@@ -0,0 +1,39 @@
+# c01-git03
+
+
+1. Did you notice any difference in the size of the repo before and after adding the big file?
+
+> Yes, before adding, the big file the size was 100K however after commit the size was increased to 20M and then after `git revert HEAD` the size remains within 10M. 
+
+2. What is the reason for this problem?
+
+> The situation occurs because git stores references needed to restore the file such as a backup in case of mis deleting.
```

Comment:
It is not exactly used as "backup" (it saves the file for that branch) but you are right, you can recover it in case you delete by accident (it could be used as backup, but not the main purpose).